examples/centralwidget/autohide.pro

```diff
@@ -2,7 +2,7 @@ ADS_OUT_ROOT = $${OUT_PWD}/../..
 
```

Comment:
This file belongs into the autohide folder example. A centralwidget.pro file is missing